packages/template-retail-react-app/app/pages/checkout/partials/shipping-multi-address.jsx

```diff
@@ -80,6 +90,61 @@ MultiShippingItemAttributes.propTypes = {
     includeQuantity: PropTypes.bool
 }
 
+// Separate component for address form to properly use hooks
+const AddressFormComponent = ({item, index, onSubmit, onCancel}) => {
```

Comment:
Can we remove the word "Component"? Every function that render jsx implies it is a component.